tox.ini

```diff
@@ -1,6 +1,5 @@
 [tox]
-envlist = {ceph_ansible,ceph_ansible2.2}-{jewel}-{centos7}-{cluster,filestore_osds_container}
-	{ceph_ansible,ceph_ansible2.3}-{luminous}-{centos7}-{cluster,filestore_osds_container,bluestore_osds_container,docker_cluster_collocation}
+envlist = {ceph_ansible,ceph_ansible2.3}-{luminous}-{centos7}-{cluster,filestore_osds_container,bluestore_osds_container,docker_cluster_collocation}
```

Comment:
Not sure about this, let's keep it around and wait for @guits input regarding the CI.

Reply:
Let's wait for @guits but if we don't build the jewel image on master, I don't see any valid reason trying to test it.

Maybe we should replace jewel by mimic instead of removing jewel ?

Reply:
@guits Any concerns with this patch ?

Reply:
@ErwanAliasr1 +1 replacing jewel by mimic.
@leseb Does that mean we should drop jewel support from ceph-ansible as well?

Reply:
@guits What should be the envlist for jewel then ?
Jewel our ceph-ansible master sounds like strange for me while it's perfect in the stable branches.

Reply:
Agree with @ErwanAliasr1 Jewel will remain in stable branch (3.0 and 3.1) where current master (future 3.2) shouldn't need to support Jewel.

Reply:
@ErwanAliasr1 @leseb that's correct

Reply:
@guits @leseb If you agree with the current state of the PR, can you approve it ?